src/helper/browserStorage.ts

```diff
@@ -68,6 +68,12 @@ export class BrowserStorage {
     store[key] = value;
     this.storage.setItem(this._storeKey, JSON.stringify(store));
   }
+
+  remove(key: string): void {
+    const store = JSON.parse(this.storage.getItem(this._storeKey) || "{}");
+    delete store[key];
+    this.storage.setItem(this._storeKey, JSON.stringify(store));
+  }
```

Comment:
Should we also add a function for deleted web browser factor additionally for ease of use?

Reply:
I am thinking add a clear which clear all related browser data

Reply:
makes sense

Reply:
There is resetStore func for clearing

Reply:
oh right, cool then good to go.